includes/wikia/services/WikiService.class.php

```diff
@@ -6,6 +6,10 @@ class WikiService extends WikiaModel {
 		'sysop'
 	];
 
+	const MODERATOR_GROUPS = [
+		'chatmoderator'
```

Comment:
There should be users with `content moderator` or `discussions moderator` permissions